### ISSUE TITLE
C_LFGList.GetPlaystyleString unnecessarily hardware protected

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#195]
The `C_LFGList.GetPlaystyleString` function added in patch 9.1.5 appears to require a hardware input if called from insecure code. Given this function appears to only return a playstyle string for display from a given playstyle/activity combination it seems a bit overbearing for this API to be restricted.

The below test case will raise an action blocked error instead of printing a playstyle string ("Standard").

```
/run C_Timer.After(0, function() print(C_LFGList.GetPlaystyleString(1, C_LFGList.GetActivityInfoTable(695))); end);
```

**Comment by siweia** [comment]
This happens to all addons that relate to world map, eg handynotes and some worldquests addons. Even occurs with ElvUI only.
Open world map, hold key shift and click on world boss world quest icon, repeat for 3 or 4 times. Then you will be blocked from creating world boss group thru premade.

**Comment by ChrisKader** [comment]
AS of 9.2.5.43630 this is still a protected function

**Comment by Meorawr** [comment]
Resolved in 11.0.2.55789.

**Comment by ChrisKader** [comment]
Thanks!

— CK
________________________________
